bot/cogs/random_cog/random_cog.py

```diff
@@ -92,5 +92,44 @@ async def ball(self, ctx, *, question):
         embed = discord.Embed(title='🎱', description= f'{random.choice(responses)}',color = Colors.ClemsonOrange)
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['slotmachine','🎰'])
+    async def slots(self, ctx):
+        """
+        A simple slot machine.
+            """
+        emojis = "🍎🍊🍐🍋🍉🍇🍓🍒"
+        a = random.choice(emojis)
+        b = random.choice(emojis)
+        c = random.choice(emojis)
+
+        blank = '⬜'
+
+        if (a == b ==c):
```

Comment:
there is a better way of doing this that also scales to N number of slots, we can use set() to test for the number of same values.